apps/docs/src/examples/extras/positional-audio/Turntable.svelte

```diff
@@ -77,14 +79,16 @@
     }
     materials: {}
   }>('/models/turntable/cover.glb')
-  let coverGeometry: BufferGeometry | undefined
-  $: if ($gltf) {
-    const coverMesh = $gltf.nodes.Cover as Mesh
-    coverGeometry = coverMesh.geometry
-  }
+  let coverGeometry: BufferGeometry | undefined = $state()
+  $effect(() => {
+    if ($gltf) {
+      const coverMesh = $gltf.nodes.Cover as Mesh
```

Comment:
```suggestion
      const coverMesh = $gltf.nodes.Cover
```

it looks like `gltf` gets its type further up in the file with the `useGLTF` function.